tracer/span_test.go

```diff
@@ -35,13 +35,13 @@ func TestSpanSetMeta(t *testing.T) {
 
 	// check the map is properly initialized
 	span.SetMeta("status.code", "200")
-	assert.Equal(len(span.Meta), 1)
 	assert.Equal(span.Meta["status.code"], "200")
 
 	// operating on a finished span is a no-op
+	nMeta := len(span.Meta)
 	span.Finish()
 	span.SetMeta("finished.test", "true")
-	assert.Equal(len(span.Meta), 1)
+	assert.Equal(len(span.Meta), nMeta)
```

Comment:
Always feeling sad when I see this, I think testify expects the "expected value" to be on the left side (first arg) and the actual value on the right side. But then, the rest of this file (and of the project) has been done the wrong way. I'm mixed. But feel free to rewrite them the other way, it's just a nitpick anyway, makes error messages clearer.